ambari_gem/ambari_client.gemspec

```diff
@@ -0,0 +1,33 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'ambari_client/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = "ambari_client"
+  spec.version       = AmbariClient::VERSION
```

Comment:
Did you add a VERSION property to the AmbariClient library and/or convert the library to a module? If not, this call won't return anything useful. Should probably move the entire library under a Module namespace and add a version property.